cosmos/dbt/graph.py

```diff
@@ -136,7 +137,9 @@ def load_via_dbt_ls(self):
             command.extend(["--select", *self.select])
         logger.info(f"Running command: {command}")
         try:
-            process = Popen(command, stdout=PIPE, stderr=PIPE, cwd=self.project.dir, universal_newlines=True)
+            process = Popen(
+                command, stdout=PIPE, stderr=PIPE, cwd=self.project.dir, universal_newlines=True, env=os.environ
+            )
```

Comment:
what's the reason for this change?

Reply:
So the user `models/schema.yml` can have references to environment variables using dbt's `env_var`.